extension.html

```diff
@@ -55,6 +55,25 @@ <h2>Introduction</h2>
 	The <code>getCoalescedEvents</code> API (similar to Android <a href="https://developer.android.com/reference/android/view/MotionEvent.html#getHistoricalX(int, int)">getHistorical</a> APIs and iOS <a href="https://developer.xamarin.com/api/member/MonoTouch.UIKit.UIEvent.GetCoalescedTouches/p/MonoTouch.UIKit.UITouch/">GetCoalescedTouches</a> API) exposes all the events that were coalesced into a single event. Coalescing events into a single event enables user agents to prevent processing of the events that don’t get a chance to update the frame content visually. Basically if the javascript handlers process the events and update the screen contents faster than refresh rate only the last update will be rendered and shown in the next frame. However, some applications like drawing apps may want to take advantage of the precise history of events as user moves the pointer on the screen to be able to draw smoother and better looking curves. This API can be used in those scenarios to get all the coalesced events from the event that was dispatched to the javascript.
       </p>
     </section>
+    <section>
+      <h2>Extensions to the Pointer Event types</h2>
+      <p>Below is the new event type that is added to the <a href="https://w3c.github.io/pointerevents/index.html#pointer-event-types">existing pointer event types</a>.</p>
+      <section>
+        <h3>The <dfn><code>pointerrawmove</code> event</dfn></h3>
+        <p>A user agent MUST <a href="https://w3c.github.io/pointerevents/index.html#firing-events-using-the-pointerevent-interface">fire a pointer event</a>
+           named <code>pointerrawmove</code> when a pointing device attribute is changed.
+           As oppose to <code>pointermove</code> which could be fired once every frame and will have coalesced event data,
+           <code>pointerrawmove</code> will get fired as soon as an attribute is changed
+           and as frequent as the javascript can handle the events.
+           Because of that the target of <code>pointerrawmove</code> events might be different from the <code>pointermove</code> events
+           due to the fact that <code>pointermove</code> events are being sent once every frame and the final position of the even
```

Comment:
pointermove isn't defined to be sent once every frame.

Reply:
You are right. This part is not yet specced. Not sure how to get around it. More on it in the other comment.